packages/http-client-js/package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "@typespec/http-client-js",
+  "version": "0.1.0",
+  "type": "module",
+  "private": true,
+  "main": "dist/src/index.js",
+  "scripts": {
+    "build-src": "babel src -d dist/src --extensions .ts,.tsx",
+    "build": "tsc -p . && npm run build-src",
+    "clean": "rimraf ./dist",
+    "lint": "eslint . --max-warnings=0",
+    "lint:fix": "eslint . --fix",
+    "watch-src": "babel src -d dist/src --extensions .ts,.tsx --watch",
```

Comment:
not for this pr but wondering if we could find a better build flow for those jsx/alloy packages so you can call a single command

Reply:
Filed https://github.com/microsoft/typespec/issues/6253